VideoLocker/src/main/java/org/edx/mobile/view/CourseUnitWebviewFragment.java

```diff
@@ -111,8 +107,23 @@ public void onReceivedError(WebView view, int errorCode,
                 hideLoadingProgress();
                 pageIsLoaded = false;
                 ViewPagerDownloadManager.instance.done(CourseUnitWebviewFragment.this, false);
-                super.onReceivedError(view, errorCode, description, failingUrl);
             }
+
+            // Restore these annotations when we upgrade our compile SDK version to Marshmallow
```

Comment:
It is correct that if you forget to add ``@Override`` at all, the compiler won't tell you you should have used it? If so, is there a linter pass or something we could add? If we had that, then once we pick up API 23, that linter pass would tell us we were missing an ``@Override`` and whoever fixed that would see this line and fix the reference to VERSION_CODES too. Otherwise we probably won't ever remember to fix this (though I guess if it's not in the way, it doesn't matter toooo much).